src/config.c

```diff
@@ -443,6 +444,26 @@ CONFIG_GETTER(getIndexingMemoryLimit) {
   return sdscatprintf(ss, "%u", config->indexingMemoryLimit);
 }
 
+// BM25STD_TANH_STRETCH
+CONFIG_SETTER(setBM25StdTanhStretch) {
+  uint64_t newStretch;
+  int acrc = AC_GetU64(ac, &newStretch, AC_F_GE1);
+  CHECK_RETURN_PARSE_ERROR(acrc);
+  if (newStretch > BM25STD_TANH_STRETCH_MAX) {
+    QueryError_SetWithoutUserDataFmt(status, QUERY_ELIMIT,
+      "BM25STD_TANH_STRETCH must be between %d and %d inclusive",
+      BM25STD_TANH_STRETCH_MIN, BM25STD_TANH_STRETCH_MAX);
+    return REDISMODULE_ERR;
+  }
+  config->BM25STD_TanhStretchFactor = newStretch;
+  return REDISMODULE_OK;
+}
+
+CONFIG_GETTER(getBM25StdTanhStretch) {
+  sds ss = sdsempty();
+  return sdscatprintf(ss, "%u", config->BM25STD_TanhStretchFactor);
```

Comment:
```suggestion
  return sdscatprintf(ss, "%ul", config->BM25STD_TanhStretchFactor);
```

Reply:
Seems that `sds` does not support that option

Reply:
😮